languages/elixir/exercises/concept/structs/.meta/design.md

```diff
@@ -0,0 +1,33 @@
+## Learning objectives
+
+- Know what a struct is
+- Know that a struct is a bare map
+- Know that structs have compile time guarantees
+- Know that structs implment the static access operator (struct.key)
+- Know that structs have fields, like maps
+- Know how to declare a struct
+- Know how to declare a struct field
+- Know how to initialize a struct field with a value
+- Know how to compare structs
+
+- Know how to enforce a field's specification on creation with `@enforce_keys` annotation
```

Comment:
I would love to see that in the exercise itself. For example, we could add a field to the struct called `nickname` (you know, like people give names to their vehicles...) and make it required (also an argument to `RemoteControlCar.new`). Then we can have a test that trying to initialize the struct without the key will raise.